examples/recipe-sourcing-contentful/src/pages/news.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { graphql } from "gatsby"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+const NewsPage = ({ data }) => (
+  <Layout>
+    <SEO title="News" />
+    <h1>News</h1>
+    {data.allContentfulNews.edges.map(({ node }) => (
```

Comment:
See notes in the review about using an unordered list here for semantics purposes.

Reply:
Done in https://github.com/gatsbyjs/gatsby/pull/17318/commits/6bf620110f66fc4a9f9a982c7c38f32d166316b2